tests/test_dicom.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy of
+# the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations under
+# the License.
+# ==============================================================================
+"""Tests for DICOM."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import pytest
+
+import tensorflow as tf
+import tensorflow_io.dicom as dicom_io  # pylint: disable=wrong-import-position
+
+
+tf.compat.v1.disable_eager_execution()
```

Comment:
@marcelolerendegui Also, disable_eager_execution() need to be placed immediately after `import tensorflow as tf` as well for the same reason.